internal/ui/table/table.go

```diff
@@ -25,6 +25,24 @@ type Table struct {
 
 var funcmap = template.FuncMap{
 	"join": strings.Join,
+	"trunc": truncateStr,
+}
+
+// Truncates the first line in `s` to the given `maxLength`
+func truncateStr(maxLen int, s string) string {
+	headline := strings.Split(s, "\n")[0]
+
+	if len(headline) > maxLen {
+		if len(headline) >= 3 {
+			headline = headline[:maxLen-3]
+		} else {
+			headline = headline[:maxLen]
+		}
+
+		headline += "..."
```

Comment:
Shouldn't the `...` only be appended when the headline is shortened to `maxLen-3`? That way the resulting string actually has the requested length.